content/backmatter.tex

```diff
@@ -340,7 +340,9 @@ \chapter{\openshmem Specification and Deprecated API}\label{sec:dep_api}
     \FUNC{SHMEM\_PUT} & 1.2 & Yes & \FUNC{SHMEM\_PUT8} or \FUNC{SHMEM\_PUT64} \\ \hline
     \FUNC{SHMEM\_CACHE} & 1.3 & Yes & (none) \\ \hline
     \_SHMEM\_* constants & 1.3 & Yes & SHMEM\_* \\ \hline
-    \hline
+%\newtext{
+    SMA\_* environment variables & 1.4 & Yes & SHMEM\_* \\ \hline
```

Comment:
We should not delay #25 either.  The responsibility for this fix should fall on this proposal, which is proposing the deprecations.